### PR TITLE
Snpsift vartype

### PR DESCRIPTION
**Pull request by tdayris-perso**
Hi,

This is a new wrapper for [SnpSift varType](http://snpeff.sourceforge.net/SnpSift.html#VariantType). Other SnpSift wrappers will follow soon after this one is accepted.

Thanks in advance for your reviews,
Thibault

### CODE REVIEW

File: bio/snpsift/varType/environment.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: snpsift
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+dependencies:
```
Comment (johanneskoester) [inline bio/snpsift/varType/environment.yaml]: Just write snpsift below, channel priorities will take case of the rest. Above, you can omit defaults, it is not needed anymore. Also, the name can be omitted and the three dashes at the top.

File: bio/snpsift/varType/test/Snakefile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+rule test_snpsift_vartype:
+    input:
+        vcf = "in.vcf"
```
Comment (johanneskoester) [inline bio/snpsift/varType/test/Snakefile]: Please remove the whitespaces around the equal, following PEP8 style.

File: bio/snpsift/varType/wrapper.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+__license__ = "MIT"
+
+from os.path import dirname
+from snakemake.shell import shell
+from snakemake.utils import makedirs
+
+log = snakemake.log_fmt_shell(stdout=False, stderr=True)
+
+extra = snakemake.params.get("extra", "")
+makedirs(dirname(snakemake.output.vcf))
```
Comment (johanneskoester) [inline bio/snpsift/varType/wrapper.py]: makedirs is done automatically by snakemake for every output file.

File: bio/snpsift/varType/environment.yaml (hunk 1 above)
Comment (tdayris-perso) [reply bio/snpsift/varType/environment.yaml]: Go it, thanks!

File: bio/snpsift/varType/test/Snakefile (hunk 2 above)
Comment (tdayris-perso) [reply bio/snpsift/varType/test/Snakefile]: Corrected

File: bio/snpsift/varType/wrapper.py (hunk 3 above)
Comment (tdayris-perso) [reply bio/snpsift/varType/wrapper.py]: It does? Nice!

**Pull request merged**